docs/source/troubleshooting.rst

```diff
@@ -21,7 +21,7 @@ to help!
    `on PyPi <https://pypi.org/project/pyquil/>`_.
 
 2. If the error appears to be authentication-related, or makes any mention of your
-   ``user_auth_token``, then please update your token following the directions at
+   user auth token, then please update your token following the directions at
    https://qcs.rigetti.com/auth/token.
```

Comment:
We should refer the user to the CLI again here. If the refresh token expires, we'll need to make sure that the CLI initiates the OAuth2 web flow.

If the user is on JupyterHub we may have to suggest `rm -r ~/.qcs` and restarting their JupyterLab server.

Reply:
OK, I'll link out to the to-be-updated CLI docs here